locales/it_IT/pcsx2_Main.po

```diff
@@ -1319,89 +1324,65 @@ msgstr "BIOS"
 msgid "Folders"
 msgstr "Cartelle"
 
-#: pcsx2/gui/Dialogs/SysConfigDialog.cpp:261
-#, c-format
-msgid "Appearance/Themes - %s"
-msgstr "Aspetto/Temi - %s"
-
-#: pcsx2/gui/Dialogs/SysConfigDialog.cpp:268
-msgid "Appearance"
-msgstr "Aspetto"
-
-#: pcsx2/gui/Dialogs/SysConfigDialog.cpp:278
-#, fuzzy
+#: pcsx2/gui/Dialogs/SysConfigDialog.cpp:260
 msgid "Language Selector"
 msgstr "Selettore Lingua"
 
 #: pcsx2/gui/ExecutorThread.cpp:41
 msgid "Logs events as they are passed to the PS2 virtual machine."
 msgstr ""
-"Mostra il log degli eventi che sono trasmessi alla macchina virtuale PS2."
+"Mostra il registro degli eventi che sono trasmessi alla macchina virtuale "
+"PS2."
 
-#: pcsx2/gui/ExecutorThread.cpp:430
-msgid "Press Cancel to attempt to cancel the action."
-msgstr "Premi Annulla per tentare di annullare questa azione."
-
-#: pcsx2/gui/ExecutorThread.cpp:431
-#, c-format
-msgid "Press Terminate to kill %s immediately."
-msgstr "Premi Termina per terminare %s immediatamente."
-
-#: pcsx2/gui/ExecutorThread.cpp:434
-msgid "Terminate App"
-msgstr "Termina applicazione"
-
-#: pcsx2/gui/FrameForGS.cpp:444
+#: pcsx2/gui/FrameForGS.cpp:448
 msgid "GS Output is Disabled!"
 msgstr "L'output GS è disabilitato!"
 
-#: pcsx2/gui/GlobalCommands.cpp:309
+#: pcsx2/gui/GlobalCommands.cpp:312
 msgid "Exit PCSX2?"
 msgstr "Uscire da PCSX2?"
 
-#: pcsx2/gui/GlobalCommands.cpp:474
-#, fuzzy
+#: pcsx2/gui/GlobalCommands.cpp:477
 msgid "Save state"
 msgstr "Salva stato"
 
-#: pcsx2/gui/GlobalCommands.cpp:475
+#: pcsx2/gui/GlobalCommands.cpp:478
 msgid "Saves the virtual machine state to the current slot."
 msgstr "Salva lo stato della macchina virtuale nello slot corrente."
 
-#: pcsx2/gui/GlobalCommands.cpp:481
-#, fuzzy
+#: pcsx2/gui/GlobalCommands.cpp:484
 msgid "Load state"
 msgstr "Carica stato"
 
-#: pcsx2/gui/GlobalCommands.cpp:482
+#: pcsx2/gui/GlobalCommands.cpp:485
 msgid "Loads a virtual machine state from the current slot."
 msgstr "Carica uno stato della macchina virtuale dallo slot corrente."
 
-#: pcsx2/gui/GlobalCommands.cpp:488
+#: pcsx2/gui/GlobalCommands.cpp:491
 msgid "Load State Backup"
 msgstr "Carica stato dal backup"
 
-#: pcsx2/gui/GlobalCommands.cpp:489
+#: pcsx2/gui/GlobalCommands.cpp:492
 msgid "Loads virtual machine state backup for current slot."
 msgstr ""
 "Carica lo stato della macchina virtuale dalla copia di sicurezza dello slot "
 "corrente."
 
-#: pcsx2/gui/GlobalCommands.cpp:495
+#: pcsx2/gui/GlobalCommands.cpp:498
 msgid "Cycle to next slot"
 msgstr "Seleziona slot successivo"
 
-#: pcsx2/gui/GlobalCommands.cpp:496
+#: pcsx2/gui/GlobalCommands.cpp:499
 msgid "Cycles the current save slot in +1 fashion!"
-msgstr "Passa allo slot di salvataggio successivo"
+msgstr "Aggiunge +1 al numero di slot di salvataggio corrente!"
```

Comment:
Scorre di +1 il numero di slot di salvataggio corrente ?

Reply:
Nella precedente traduzione non ero riuscito a rendere in italiano questa spiritosa descrizione.
Adesso ci ho provato, ma sembra che non abbia raggiunto l'obiettivo.
L'alternativa che mi hai fornito va bene. Potrei inoltre provare a cambiare "corrente" con "selezionato".

Reply:
Eccetto che non necessariamente hai selezionato uno slot /s

Reply:
Che io sappia di default viene selezionato lo slot 0 per i salvataggi di stato e quando si cambia utilizzando F2 nella console o nell'OSD viene visualizzato "selected savestate slot #". Il mio ragionamento partiva da questo. Ma comunque "corrente", va bene.
Quindi finalizzerò così:
`Scorre di -1 il numero di slot di salvataggio corrente!`
`Scorre di +1 il numero di slot di salvataggio corrente!`

PS: non ho mai visto/capito dove viene utilizzata questa stringa, credo che sia una delle descrizioni dei tasti di scelta rapida e attualmente non sia utilizzata.